From f17b5e5be37875e05794621eacf386ec25adf223 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Mon, 26 Oct 2015 14:07:55 +0800
Subject: [PATCH] MLK-11758 dts: mx6ul evk: set regulator wlreg_on always on

Murata adapter VDDIO is derived from sd slot VMMC supply, if usdhc cannot
detect sd/sdio card, VMMC is shutdown that causes VDDIO has no voltage, and
bt cannot work. So BT depends on VDDIO/VMMC.

Now the workaround is set wlreg_on alwasys on, usdhc can detect wifi card
and supply VMMC 3.3V voltage, so that Murata adapter VDDIO has power supply.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
index 51e8bbd..1f3f8d5 100644
--- a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
+++ b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
@@ -21,6 +21,7 @@
 			regulator-name = "wlreg_on";
 			gpio = <&gpio5 1 0>;
 			startup-delay-us = <100>;
+			regulator-always-on;
 			enable-active-high;
 		};
 	};
-- 
1.8.0

